tracim/tracim/lib/workspace.py

```diff
@@ -3,6 +3,7 @@
 
 from sqlalchemy.orm import Query
 from tg.i18n import ugettext as _
+from typing import List
```

Comment:
common usage is to import typing, so that there is only one import for all typing stuff (and then in code you write `typing.List[Workspace]` instead of `List[Workspace]`